src/CmsBundle/Security/AccessVoter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Opifer\CmsBundle\Security;
+
+use Opifer\CmsBundle\Entity\User;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+use Symfony\Component\Security\Core\Security;
+
+class AccessVoter extends Voter
```

Comment:
I'd rename this to something like `ActionVoter` or something similar. Since every voter votes to grant `access`